doc/source/config-files.rst

```diff
@@ -19,11 +19,9 @@ If you are planning to submit jobs to a cluster, then you need to know about a s
 Pipeline developers
 *****************
 
-If you want to add a new pipeline to looper, then there are two YAML files that coordinate linking your pipeline in to your looper project.
+If you want to add a new pipeline to looper, then you need to know about a configuration file that coordinates linking your pipeline in to your looper project.
 
--   :doc:`protocol mapping file <protocol-mapping>`: Tell looper which pipelines exist, and how to map each library (sample data type) to its pipelines.
-
--	:doc:`pipeline interface file <pipeline-interface>`: Links looper to the pipelines, describes variables, options and paths that the pipeline needs to know to run, and resource requirements for cluster managers.
+-	:doc:`pipeline interface file <pipeline-interface>`: Has two sections: ``protocol_mapping`` looper which pipelines exist, and how to map each library (sample data type) to its pipelines. ``pipelines`` links looper to the pipelines, describes variables, options and paths that the pipeline needs to know to run, and resource requirements for cluster managers.
```

Comment:
Looks like maybe there should be something like "tells" here (`protocol_mapping` "tells looper...")? Also, do you want to replace "library" with "protocol" here since that's the direction things seem to be heading?